gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.        ax.plot_surface(X, Y, Z, rstride=3, cstride=3, color='white', edgecolor="gray")
</pre><p>I like the shiny style in the original plot. Can you please use that? maybe check Python data-science handbook for API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>